src/components/dropdown/dropdown.js

```diff
@@ -382,7 +382,7 @@ const Dropdown = Input(InputIcon(InputLabel(InputValidation(class Dropdown exten
     let nextVal;
 
     switch (ev.which) {
-      case 13: // return
+      case 9: case 13: // tab key or return key
```

Comment:
💄 `case`'s should be on separate lines always for readability
but what concerns me more is that we're preventing default behaviour of "tab" which is going to the next element - we're basically forcing people to press tab twice per dropdown, which makes it less smooth experience than just tabbing straight away to the next - IMO in case of tab default action should not be prevented, I quickly checked it and not doing it gives the experience I'd expect from tabbing (needs investigation of possible side effects but not sure there will be any)